main.py

```diff
@@ -14,7 +14,7 @@ def main():
     with open('números.txt', 'r', encoding='utf8') as arquivo:
         linhas_do_arquivo = arquivo.readlines()
 
-    # Pegue a lista de strings e converta todos os valores dentro dela para inteiros.
+        # Pegue a lista de strings e converta todos os valores dentro dela para inteiros.
     # TODO COLOQUE SEU CÓDIGO AQUI E APAGUE ESSE COMENTÁRIO DEPOIS.
 
```

Comment:
Aqui está o problema de não conseguir testar com o arquivo texto que eu coloquei no repositório.
Precisa pegar a lista de strings e transformar ela em uma lista de números inteiros.
```suggestion
    lista_numeros = map(int, linhas_do_arquivo)
```